functions/v2/http-logging/index.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START functions_structured_logging]
+const {Logging} = require('@google-cloud/logging');
+const functions = require('@google-cloud/functions-framework');
+const pkg = require('./package.json');
+
+functions.http('structuredLogging', async (req, res) => {
+  // Initialize the logging client
+  const logging = new Logging();
+  // Required to capture your project id
+  await logging.setProjectId();
+  // Create a LogSync transport, defaulting to process.stdout
+  const log = logging.logSync(pkg.name);
```

Comment:
Nit: use a variable, instead of package name?

Reply:
Edit: disregard, see [Averi's comment here](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/2596#discussion_r851429858).